src/main/java/com/yugabyte/sample/apps/AppBase.java

```diff
@@ -572,6 +574,7 @@ public void dropCassandraTable(String tableName) {
    * @throws java.lang.Exception in case of CREATE statement errors.
    */
   public void createTablesIfNeeded(TableOp tableOp) throws Exception {
+    LOG.info("Creating Cassandra tables...");
```

Comment:
Can we please change from `Cassandra` to `YCQL` when logging.

Reply:
Sure